astropy/table/table.py

```diff
@@ -2021,6 +2021,42 @@ def group_by(self, keys):
 
         return groups.table_group_by(self, keys)
 
+    def to_pandas(self):
+        """
+        Return a :class:`pandas.DataFrame` instance
+
+        Returns
+        -------
+        dataframe : :class:`pandas.DataFrame`
+            A pandas :class:`pandas.DataFrame` instance
+        """
+        from pandas import DataFrame
+        print(self.has_mixin_columns)
+        if self.has_mixin_columns:
+            raise ValueError("Cannot convert a table with mixin columns to a pandas DataFrame")
+        for column in self.columns:
```

Comment:
I think this can be a bit cleaner with `for column in self.columns.values():` then check `column.ndim > 1`.  If you like functional, `if any(col.ndim > 1 for col in self.columns.values()):` is even better.